main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -738,6 +741,8 @@ private void notifyDataSetChanged() {
             return;
         }
 
+        final ScrollView scroll = ButterKnife.findById(this, R.id.detailScroll);
```

Comment:
good question - I can move this 17 lines down, right before 
reinitializeViewPager();
Can't execute this after this call because I would get the new scrollview.